pos_hide_partner_info/static/src/xml/templates.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template">
+    <t t-extend="ClientListScreenWidget">
+
+        <t t-jquery=".client-list" t-operation="replace">
+
+            <table class='client-list'>
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                    </tr>
+                </thead>
+                <tbody class='client-list-contents'>
+                </tbody>
+            </table>
+
+        </t>
+    </t>
+
+    <t t-extend="ClientLine">
+        <t t-jquery=".client-line" t-operation="replace">
+            <tr class='client-line' t-att-data-id='partner.id'>
+                <td>
+                    <t t-esc='partner.name' />
+                </td>
+            </tr>
+        </t>
+    </t>
+
+    <t t-extend="ClientDetails">
+        <t t-jquery=".client-details-left" t-operation="replace">
+            <div class='client-details-left' />
```

Comment:
sale remark as https://github.com/OCA/pos/pull/879

For exemple, this module can conflict with
-  ``pos_partner_firstname`` (``<t t-jquery=".client-details-left" t-operation="prepend">``)
- ``pos_customer_required_fields`` (``<t t-jquery="div.client-details-left" t-operation="append">``)

Reply:
I don't see a way to select [these `client-detail` elements](https://github.com/odoo/odoo/blob/c53081f10befd4f1c98e46a450ed3bc71a6246ed/addons/point_of_sale/static/src/xml/pos.xml#L346)  that have a child with class `client-address-street`. Apparently it's not possible to select the parent of an element with CSS selectors. 
Is it OK if I make the whole [client-details-left](https://github.com/odoo/odoo/blob/c53081f10befd4f1c98e46a450ed3bc71a6246ed/addons/point_of_sale/static/src/xml/pos.xml#L345) invisible, with a warning in Usage.rst ?

Reply:
@victor-champonnois I don't think that would work. We will actually use both `pos_partner_firstname` and `pos_hide_partner_info` on our production server :/

Reply:
OK I found ways to do it without brute force replacing.